src/styles/functions/_max-width.scss

```diff
@@ -4,8 +4,10 @@
   $value-type: type_of($value);
 
   @if $value-type != number {
-    @warn "Expected $value #{$value} [#{$value-type}] to be of type Number.";
+    // @warn "Expected $value #{$value} [#{$value-type}] to be of type Number.";
 
+    // node-sass has a --quiet flag but it's not respected (all scss @warn statements like the one above are still printed)
```

Comment:
Lets fix this instead then

Reply:
I could not find it where.

Reply:
Okay, so let's [update `node-sass`](https://github.com/sass/node-sass/releases/tag/v4.11.0) first.
If the [`--quite` option](https://github.com/sass/node-sass#usage-1) is still broken, we should create an issue over there.

Nevertheless since `quite` is a boolean option it should work:
http://sass-lang.com/documentation/file.SASS_REFERENCE.html#options

Reply:
In regard to rollup, all sass options have to namespace:
https://www.npmjs.com/package/rollup-plugin-sass#options-1

Reply:
Does not work.

Reply:
If it's really not working, please file an issue over there

Reply:
I will.